src/components/header/header.stories.tsx

```diff
@@ -15,10 +15,19 @@ const Template: ComponentStory<typeof Header> = (args) => <Header {...args} />;
 export const Regular = Template.bind({});
 Regular.args = {
   siteTitle: 'Satellytes',
+  $lightTheme: false,
+  showLanguageSwitch: true,
 };
 
-export const WithLanguageSwitch = Template.bind({});
-WithLanguageSwitch.args = {
+export const WithLightTheme = Template.bind({});
+WithLightTheme.args = {
   ...Regular.args,
-  showLanguageSwitch: true,
+  $lightTheme: true,
+};
+
+export const WithLightThemeTransparent = Template.bind({});
+WithLightThemeTransparent.args = {
+  ...Regular.args,
+  $lightTheme: true,
+  transparent: true,
```

Comment:
Thanks for your review 👍
I have the same problems with the transparent property in the storybook. I did a bit of research and tried a few things (e.g. define custom argTypes), however, it still didn't work.
That's why I would also suggest leaving the three stories as they are, so you can see all styles. In general the approach "fewer stories with many controls" sounds good. I will include it in other PRs

Reply:
I had a look and figured out it's because of the way the initial state of `isHeaderTransparent` is set up. It takes the initial value of `props.transparent` during the mounting and from then on ignores any value changes of `props.transparent`. Let's keep it as it is and maybe work on this in the future. It's not going to be a problem on the actual website.

Reply:
> it seems like props.transparent is only taken into account during the initial mounting of the component but later changes are ignored. This is nothing that you introduced so no need to fix it in this PR. Maybe you want to have a look anyways?

I have checked the header component, it's missing an `useEffect` to update the property. It's counter intuitive but the default value for useState is not updated on future renderings. That's `useEffect` for, where you can tell what is going to change and then trigger the state update.

With the following code in header it will work. Please provide a fix after merging this PR 👍

```
const transparent = Boolean(props.transparent);

useEffect(() => {
    setIsHeaderTransparent(transparent)
  }, [transparent])
```